src/example-types/filters/tagsQuery.js

```diff
@@ -80,6 +80,50 @@ let filter = ({ tags, join, field, exact }) => ({
   },
 })
 
+let result = async (node, search, schema, { options, getProvider }) => {
+  let { tags, join, field, exact, _meta, maxToCount = 20 } = node
+  if (_.size(tags) > maxToCount) {
+    return null
+  }
+
+  let provider = getProvider(node)
+
+  let results = await Promise.all(
+    _.map(
+      async tag => ({
+        tag,
+        result: await provider.runSearch(
+          options,
+          node,
+          schema,
+          provider.groupCombinator({ join: 'and' }, [
+            _meta.relevantFilters,
+            {
+              query_string: {
+                query: tagsToQueryString([tag], join),
+                default_operator: 'AND',
+                default_field:
+                  field.replace('.untouched', '') + (exact ? '.exact' : ''),
+                ...(exact && { analyzer: 'exact' }),
+              },
+            },
```

Comment:
I think you can reuse the `filter` method in this file

Reply:
also no longer relevant